site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -448,13 +460,17 @@ user_registration_request = {
 Finally, let's specify our `client_response`.
 
 ```python
-client_response = client.register(None, user_registration_request)
+client_response = client.register(user_registration_request, None)
 if client_response.was_successful():
     print(client_response.success_response)
 else:
     sys.exit(client_response.error_response)
 ```
 
+{% include _callout-important.liquid content=
+"If you are running a version of the FusionAuth python client 1.15.3 or earlier, use ```client.register(None, user_registration_request)```"
```

Comment:
Love the call out. Please add a period at the end.

Reply:
fixed in latest commit